src/main/java/seedu/address/model/event/Event.java

```diff
@@ -2,11 +2,16 @@
 
 import static seedu.address.commons.util.CollectionUtil.requireAllNonNull;
 
+import java.time.LocalDate;
+import java.time.LocalTime;
+import java.time.format.DateTimeFormatter;
+import java.util.ArrayList;
```

Comment:
should have newline to separate java.time and java.util

Reply:
Ok done